dts/bindings/i2c/i2c-device.yaml

```diff
@@ -10,23 +10,15 @@ version: 0.1
 description: >
     This binding gives the base structures for all i2c devices
 
+inherits:
+    !include base.yaml
+
 parent:
     bus: i2c
 
 properties:
-    compatible:
-      type: string
-      category: required
-      description: compatible strings
-      generation: define
     reg:
-      type: array
-      description: address on i2c bus
```

Comment:
We may want to keep this description override for i2c, spi, etc. bus addresses

Reply:
> We may want to keep this description override for i2c, spi, etc. bus addresses

Was going to clean up the reg for buses in a follow up.  Will add back the description than.